js/co-authors-plus.js

```diff
@@ -389,6 +396,7 @@ jQuery( document ).ready(function () {
 		var $post_coauthor_names = jQuery( 'input[name="coauthorsinput[]"]' );
 		var $post_coauthor_emails = jQuery( 'input[name="coauthorsemails[]"]' );
 		var $post_coauthor_nicenames = jQuery( 'input[name="coauthorsnicenames[]"]' );
+		var $post_coauthoravatar = jQuery( 'input[name="coauthorsavaters[]"]' );
```

Comment:
Not a big deal, but can we fix the typo here of `avatar`? It should be `coauthorsavatars`.